src/schnetpack/representation/schnet.py

```diff
@@ -120,9 +127,28 @@ def __init__(
         self.radial_basis = radial_basis
         self.cutoff_fn = cutoff_fn
         self.cutoff = cutoff_fn.cutoff
-
-        # layers
-        self.embedding = nn.Embedding(max_z, self.n_atom_basis, padding_idx=0)
+        self.activate_charge_spin_embedding = activate_charge_spin_embedding
+
+        # nuclear embedding layer (often complex nuclear embedding has negative impact on the performance of the model, so we can use simple nuclear embedding to avoid this issue)
+        if nuclear_embedding != "simple":
```

Comment:
the if cases are a bit clunky. It would be nice if we could have arguments with nuclear_embedding and electronic_embedding instead. Same for the painn class

Reply:
done in c86b404 and f99a432